docs/content-services/components/search-date-range-advanced.component.md

```diff
@@ -0,0 +1,57 @@
+
+```html
+<adf-search-date-range-advanced></adf-search-date-range-advanced>
+```
+
+## Class Members
+
+### Properties
+
+| Name         | Type                    | Description                                                                                                                                                                                                                                                |
+|--------------|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| field        | string                  | Field to apply the query to. Required value                                                                                                                                                                                                                |
+| maxDate      | string                  | A fixed date (default format: dd-MMM-yy) or the string `"today"` that will set the maximum searchable date. Default is today.                                                                                                                              |
+| dateFormat   | string                  | Date format. Dates used by the datepicker are Javascript Date objects, using [date-fns](https://date-fns.org/v2.30.0/docs/format) for formatting, so you can use any date format supported by the library. Default is 'dd-MMM-yy (sample date - 07-Jun-23) |
+| initialValue | SearchDateRangeAdvanced | Initial value for the component                                                                                                                                                                                                                            |
+
+### Events
+
+| Name                | Type                                                                                           | Description                                                                                                                                    |
+|---------------------|------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| updatedQuery        | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<string>`                           | Emitted whenever a change is made in the component values. Emits the updated query based on the change made.                                   |
+| updatedDisplayValue | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<{ [key: string]: string }>`        | Emitted whenever a change is made in the component values. Emits the displayLabel based on the changes made                                    |
```

Comment:
type for this field seems incorrect - in ts file it is EventEmitter<string>

Reply:
Updated type in documentation to EventEmitter<string>